pkg/ingress/controller/controller.go

```diff
@@ -66,6 +67,12 @@ const (
 	// LabelNodeRoleMaster specifies that a node is a master
 	// It's copied over to kubeadm until it's merged in core: https://github.com/kubernetes/kubernetes/pull/39112
 	LabelNodeRoleMaster = "node-role.kubernetes.io/master"
+
+	// IngressAnnotationInternal is the annotation used on the Ingress
+	// to indicate that we want an internal loadbalancer service so that octavia-ingress-controller won't associate
+	// floating ip to the load balancer VIP.
+	// Default to true.
+	IngressAnnotationInternal = "octavia.ingress.kubernetes.io/internal"
```

Comment:
@lingxiankong it needs a doc update?

Reply:
+1 for a document change

Reply:
Hi, @adisky @jichenjc, i was planning to update the whole octavia-ingress-controller doc in a separate PR after this PR and #502 merged. Stay tuned :-)

Reply:
@lingxiankong thanks!